saleor/order/views.py

```diff
@@ -90,6 +90,8 @@ def start_payment(request, order, variant):
         try:
             form = payment.get_form(data=request.POST or None)
         except RedirectNeeded as redirect_to:
+            # NOTE: this is good place to add logic of sending
+            # digital products. Make sure that payment status is confirmed
```

Comment:
This exception is thrown only when form was processed. After payment changes status to `confirmed` or `preauth` we are redirected to `order_detail` view. I don't think that `order_detail` is a good place to ship product because user can visit it many times. IMO best place to ship product is right after payment confirmation and before any other action so that is the best place. But we can just delete this note and let other decide where they want to put logic :)